examples/anomaly_detection.py

```diff
@@ -100,6 +100,7 @@
         time_index = pd.period_range(
             data_entry["forecast_start"][i],
             periods=context_length + prediction_length,
+            freq=data_entry["forecast_start"][i].freq,
```

Comment:
Boy looks this ugly. But it is what it is.

Reply:
👹